paper.tex

```diff
@@ -379,7 +404,7 @@ \subsection{Scenario Description}
         \hline                       
         Cycle length (months)             & 15        & 18                & 12        & 15 \\
         Refueling outage (months)         & 3         & 0                 & 3         & 0 \\
-        Batch size ($\text{kg} \cdot \text{HM}$) & 29,565    & 29,565            & 8,025     & 40,125 \\
+        Batch size ($\text{kg} \cdot \text{HM}$) & 29,565    & 88,695            & 8,025     & 40,125 \\
```

Comment:
Actually, the typo is in the SFR side - the batch size is 8025 for fleet and indiv cases and LWR side should stay the same.